patches/minecraft/net/minecraft/block/Block.java.patch

```diff
@@ -207,6 +221,33 @@
 +    protected ThreadLocal<EntityPlayer> harvesters = new ThreadLocal();
 +    private ThreadLocal<IBlockState> silk_check_state = new ThreadLocal();
 +    protected static java.util.Random RANDOM = new java.util.Random(); // Useful for random things without a seed.
++
++    /**
++     * Returns whether or not this block is of a type that needs random ticking.
++     * Called for ref-counting purposes by ExtendedBlockStorage in order to
++     * broadly cull a chunk from the random chunk update list for efficiency's sake.
++     *
++     * @param state The current block state
++     * @return True if the block should tick randomly
++     */
```

Comment:
I'd recommend to:
* change the `@return` tag to what is mentioned at the top
* link to the EBS class
* remove the `@param state The current block state` line altogether since it adds no useful information and just takes up space.

With my recommendations it would look like the following in the source: 
```java
/**
 * Called for ref-counting purposes by {@link net.minecraft.world.chunk.storage.ExtendedBlockStorage} in order to broadly cull a chunk from the random chunk update list for efficiency's sake.
 * @return Whether or not this block is of a type that needs random ticking.
 */
```